BLSQ/iaso
# Ia 2328 remove countries from dropdown

Dropdown options in LQAs/Im screens showed countries for which there was no config. 

Related JIRA tickets : IA-2328

## Self proofreading checklist


## Changes

- Add an endpoint that returns countries with configs for lqas or im, and formats the response to be directly usable by the front-end
- Modify front-end accordingly

## How to test

Go to LQAs then IM
In each case check the countries in the select box. The list shouldn't contain all countries in the datasource, only the countries for which there is a config in the DB

## Print screen / video

![Screenshot 2023-07-19 at 16 33 38](https://github.com/BLSQ/iaso/assets/38907762/d43d1596-745a-4d73-a06a-1513d09f9915)
